wolox-books/src/views/Register.vue

```diff
@@ -146,7 +146,10 @@ export default {
 }
 
 .secondary-button {
+  align-items: center;
+  display: flex;
```

Comment:
Create a class 'center' outside this view in 'src/scss/commons/display.scss' and use that class in the pug code. Don't forget to add the new scss file to 'application.scss'